### PR TITLE
DCC Consent 2 screen screenshots (EXPOSUREAPP-10489)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new

Screenshot for DCC Ticketing Consent 2 screen
DccTicketingConsentTwoFragment

## Jira
[Jira](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10652)
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10195)

## How to test
Find and run `DccTicketingConsentTwoFragmentTest`

### CODE REVIEW

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    private fun generateDccTicketingQrCodeData(): DccTicketingQrCodeData {
+        return DccTicketingQrCodeData(
+            protocol = "protocol",
+            protocolVersion = "protocol_version",
+            serviceIdentity = "service_identity",
+            privacyUrl = "http://very_privacy_url.de",
+            token = UUID.randomUUID().toString(),
+            consent = "Yes, please",
+            subject = UUID.randomUUID().toString(),
+            serviceProvider = "Service Provider"
```
Comment (mtwalli) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt:152]: Please use the text from figma? to have the same exact one at least in German screenshots

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        )
+    }
+
+    private fun generateDccTicketingAllowListEntry(): DccTicketingAllowListEntry {
+        return DccTicketingAllowListEntry(
+            serviceProvider = "Allow List Provider",
```
Comment (mtwalli) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt:158]: same here

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            DccTicketingConsentTwoViewModel.UiState(
+                dccTicketingTransactionContext = dccTicketingTransactionContext,
+                certificate = mockTestCertificate("Andrea Schneider")
+            )
+        )
+        launchFragmentInContainer2<DccTicketingConsentTwoFragment>(
+            testNavHostController = navController,
+            fragmentArgs = fragmentArgs
+        )
+        takeScreenshot<DccTicketingConsentTwoFragment>()
```
Comment (mtwalli) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt:111]: please scroll and take other anther screenshot?

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+import testhelpers.Screenshot
+import testhelpers.launchFragmentInContainer2
+import testhelpers.takeScreenshot
+import java.util.UUID
+
+@RunWith(AndroidJUnit4::class)
+class DccTicketingConsentTwoFragmentTest : BaseUITest() {
+
+    @MockK lateinit var viewModel: DccTicketingConsentTwoViewModel
+    private val vc = DccTicketingValidationCondition(
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt:44]: is `vc` used anywhere?

**Review by jurajkusnier — COMMENTED** [review]
The certificate data are actually different then in figma

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+    @MockK lateinit var viewModel: DccTicketingConsentTwoViewModel
+
+    private val dccTicketingTransactionContext: DccTicketingTransactionContext = DccTicketingTransactionContext(
+        initializationData = generateDccTicketingQrCodeData(),
+        allowlist = setOf(generateDccTicketingAllowListEntry())
+    )
+
+    private val fragmentArgs = DccTicketingConsentTwoFragmentArgs(
+        containerId = TestCertificateContainerId("testCertificateContainerId")
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/consent/two/DccTicketingConsentTwoFragmentTest.kt:53]: Figma has a vaccination certificate

**Review by jurajkusnier — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**